internal/dataplane/parser/translate_grpcroute.go

```diff
@@ -62,7 +63,8 @@ func (p *Parser) ingressRulesFromGRPCRoute(result *ingressRules, grpcroute *gate
 		if p.featureFlags.ExpressionRoutes {
 			routes = translators.GenerateKongExpressionRoutesFromGRPCRouteRule(grpcroute, ruleNumber)
 		} else {
-			routes = translators.GenerateKongRoutesFromGRPCRouteRule(grpcroute, ruleNumber)
+			prependRegexPrefix := p.kongVersion.GTE(versions.ExplicitRegexPathVersionCutoff)
+			routes = translators.GenerateKongRoutesFromGRPCRouteRule(grpcroute, ruleNumber, prependRegexPrefix)
```

Comment:
```suggestion
			routes = translators.GenerateKongRoutesFromGRPCRouteRule(grpcroute, ruleNumber, p.featureFlags.RegexPathPrefix)
```
It's already precalculated.

Reply:
fixed.